### DISCUSSION TITLE
如何在有内容安全策略的网页加载Eruda

### DISCUSSION THREAD

**Discussion by 54145a** [liriliri/eruda#438] [Q&A] [answered]
如题，我想在有内容安全策略的网页加载eruda，我用的是安卓版Edge，可以在地址栏执行代码，但是无法粘贴太长的代码比如完整的eruda就不行。connect-src使我无法通过fetch获取eruda，script-src使我无法通过script元素加载eruda。

**Comment by MasterInQuestion** [comment, marked as answer]
Enhanced loader for Eruda
    [ https://github.com/liriliri/eruda/issues/327](https://github.com/liriliri/eruda/issues/327)
